docs/configuration/radio/security.mdx

```diff
@@ -127,3 +127,26 @@ ALL Security config options are available in the Web client, but at present only
 
 </TabItem>
 </Tabs>
+
+## Security Keys - Backup and Restore
+
+ ### Backup
+
+ Backing up your security keys (private and public) are vital to the long term sustainability of your node on the mesh. Once your node is on the mesh with a set of security keys, it is your responsibility to ensure these keys are maintained.
+
+ These keys can be saved in a multitude of different ways and should be stored securely.
+
+ Available ways to save these keys
+
+ 1. `meshtastic --export-config > config_backup.yaml` - this is the perferred way to backup configuration, it ensures the keys are saved intact
```

Comment:
```suggestion
 1. With the [Meshtastic CLI](/docs/software/python/cli/), run: `meshtastic --export-config > config_backup.yaml` - this will save your entire configuration including your public and private keys to a file.
```